plugins/power/tasmota_

```diff
@@ -0,0 +1,155 @@
+#!/bin/sh
+#
+# Copyright (C) 2019-2022 Tim Connors
```

Comment:
Please use the `perldoc`-style documentation format (see one of the other plugins - look for `=cut`).
This improves the rendering in the plugin gallery and is used for `munindoc PLUGIN`.

Reply:
Done